src/Player.js

```diff
@@ -39,23 +39,22 @@ class Player extends Character {
 
   /**
    * Convert prompt tokens into actual data
-   * @param {string} prompt
-   * @param {object} data
+   * @param {function} promptBuilder (data) => prompt template
+   * @param {object} extraData (key is used by promptBuilder)
    */
-  interpolatePrompt(prompt, data) {
-    data = Object.assign(
-      data || {},
-      this.getAttributes()
-    );
-
-    for (const key in data) {
-      prompt = prompt.replace('%' + key, data[key]);
-    }
-
-    // replace any unknown tokens
-    prompt = prompt.replace(/%([a-z_]+?)/, '[BAD: $1]');
-
-    return prompt;
+  interpolatePrompt(promptBuilder, extraData = {}) {
+    const buildAttributeStr = attr => `${this.getAttribute(attr)}/${this.getMaxAttribute(attr)}`;
```

Comment:
Gonna have to revisit this later, this change makes the prompt a lot less flexible because it only allows for health/energy. Will work on this tonight to bring back the token approach